src/main/java/com/elroykane/usermanangement/exception/NoUserFoundException.java

```diff
@@ -0,0 +1,12 @@
+package com.elroykane.usermanangement.exception;
+
+public class NoUserFoundException extends RuntimeException 
+{
+
+	private static final long serialVersionUID = 1L;
```

Comment:
Hey Jain. I've never really understood what serialVersionUID is for. Everytime I use eclipse, I get a warning to use for some reason🤷🏾‍♂️

Reply:
I have seen this article on the javadoc: https://docs.oracle.com/javase/6/docs/api/java/io/Serializable.html
"The serialization runtime associates with each serializable class a version number, called a serialVersionUID, which is used during deserialization to verify that the sender and receiver of a serialized object have loaded classes for that object that are compatible with respect to serialization. If the receiver has loaded a class for the object that has a different serialVersionUID than that of the corresponding sender's class, then deserialization will result in an InvalidClassException. A serializable class can declare its own serialVersionUID explicitly by declaring a field named "serialVersionUID" that must be static, final, and of type long"

You can also refer: http://www.javapractices.com/topic/TopicAction.do?Id=45